notification/src/server.js

```diff
@@ -7,8 +7,8 @@ const logger = require('./utils/logger').getLogger()
 function setupServer(routes = defaultRoutes) {
   return http.createServer(async (request, response) => {
     const parts = url.parse(request.url)
-    const route = routes[parts.pathname]
-
+    const path = parts.path?.split('/')?.slice(-2)?.[0] ?? ''
```

Comment:
@lojzatran would be nice to have it in a function with a name, which explain what is going on. 🙃

Reply:
Welcome back @LEQADA, thanks for the comment. I made a change:

https://github.com/commercetools/commercetools-adyen-integration/pull/953/commits/fd8bc8125162136f3665ee33952e3e740c5eae71

Reply:
So awesome! So fast! Thank you!